### ISSUE TITLE
Example code, channels enabled

### ISSUE THREAD

**Issue by MasterJubei** [tspspi/pymso5000#1]
Hi, thanks for making this.

When I tried to run the example code, I get:

```python
plt.plot(data['x'], data['y0'], label = "Ch1")
                        ~~~~^^^^^^
KeyError: 'y0'
```

In the example code you have:

```python
from pymso5000.mso5000 import MSO5000

with MSO5000(address = "10.0.0.123") as mso:
   print(f"Identify: {mso.identify()}")

   mso.set_channel_enable(1, True) # This should be channel 0 right?
   mso.set_channel_enable(2, True) # And this should be channel 1?

   data = mso.query_waveform((1, 2)) # Same thing here, 0 and 1
   print(data)

   import matplotlib.pyplot as plt
   plt.plot(data['x'], data['y0'], label = "Ch1")
   plt.plot(data['x'], data['y1'], label = "Ch2")
   plt.show()
```

I added some comments on the mso.set_channel_enable lines
Those should be 0 indexed right? Not 1 indexed?

Thanks

**Comment by tspspi** [comment]
True thanks for pointing that out - this is a remnant from the time where I implemented this only for the MSO5000 without the pylabdevs baseclass that uses 0 based indexing and just passed the raw values supplied by the application to the MSO.

**Comment by tspspi** [comment]
Quickly fixed in ab19d23ef85a1ed15d6c5be2648ebf33c870138b
